node/test/util/add.js

```diff
@@ -139,7 +139,7 @@ a=B|x=S:C2-1 a/b=Sa:1;Oa/b W x/y/z=a,x/r/z=b;Bmaster=2`,
                 }
                 const doAdd = co.wrap(function *(repos) {
                     const repo = repos.x;
-                    yield Add.stagePaths(repo, c.paths, stageMeta);
+                    yield Add.stagePaths(repo, c.paths, stageMeta, false);
```

Comment:
Please make the value of this flag come from the test case, and add a couple of cases that exercise it.